### PR TITLE
Improve fixed alerts

### PR DESCRIPTION
**Pull request by joshsmith**
# What's in this PR?

WIP spike of improving the fixed alerts, and adding them to alert us of behaviors on the task board.

### CODE REVIEW

**Comment by joshsmith** [comment]
This needs some work to make it so the flash messages can stack vertically from the bottom left corner of the screen.

File: app/templates/components/flash-messages/fixed-position.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{#each flashMessages.queue as |flash|}}
```
Comment (begedin) [inline app/templates/components/flash-messages/fixed-position.hbs]: Not critical for this PR to be mergeable, but could we not use a filter here?

Comment (joshsmith) [reply app/templates/components/flash-messages/fixed-position.hbs]: You mean to filter-by flash.fixed in this case and reject-by in the other?

Comment (begedin) [reply app/templates/components/flash-messages/fixed-position.hbs]: Yes, that's what I had in mind.

**Pull request merged**